### PR TITLE
update plugin to pass the proper rave parameters

### PR DESCRIPTION
**Pull request by anjolabassey**
- update plugin to pass the proper rave parameters
- implement same base url
- update readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,17 +27,31 @@ npm install vue vue-ravepayment --save
...
+       :metadata="meta"
+       :subaccounts="subaccounts"
+       :redirectUrl="redirect"
+       :paymentPlan="plan"
+       :customerFirstname="fname"
+       :customerLastname="lname"
+       paymentOptions="card,barter,account,ussd"
+       hostedPayemt=1
+       customTitle="Testing title"
+       customLogo="https://www.freelogodesign.org/Content/img/logo-samples/bakary.png"
```
Comment (iamraphson) [inline README.md]: `customLogo` is not compulsory so please kindly remove it

Comment (anjolabassey) [reply README.md]: @iamraphson yes it is not compulsory, this is just a sample for users. same for the other parameters that are not compulsory but documented in the readme.

Comment (temiadesina) [reply README.md]: Hi @iamraphson bumping this up, thanks!

Comment (iamraphson) [reply README.md]: @temiadesina no need for the default.

Comment (anjolabassey) [reply README.md]: @iamraphson i've taken it out. kindly merge

Comment (temiadesina) [reply README.md]: Hi @iamraphson please bumping this up, thanks!

**Pull request merged**